auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -34,25 +34,27 @@ class CheckPullRequest extends AuthenticatedRequestHandler {
   final ApproverServiceProvider approverProvider;
 
   static const int kPullMesssageBatchSize = 100;
+  static const int kPubsubPullNumber = 5;
 
   @override
   Future<Response> get() async {
     final Set<int> processingLog = <int>{};
-    final pub.PullResponse pullResponse = await pubsub.pull('auto-submit-queue-sub', kPullMesssageBatchSize);
     final ApproverService approver = approverProvider(config);
-    final List<pub.ReceivedMessage>? receivedMessages = pullResponse.receivedMessages;
-    if (receivedMessages == null) {
+    final List<pub.ReceivedMessage> messageList = await pullMessages();
+    if (messageList.isEmpty) {
```

Comment:
I'd recommend removing this check. Sometimes even though there are messages in the queue a call to the api return empty results.

Reply:
We want to exit early if no message is returned. Updated the message to `No messages are pulled.` to avoid confusion.